cdc/api/v2/model.go

```diff
@@ -788,6 +794,7 @@ type SinkConfig struct {
 	KafkaConfig                      *KafkaConfig        `json:"kafka_config,omitempty"`
 	MySQLConfig                      *MySQLConfig        `json:"mysql_config,omitempty"`
 	CloudStorageConfig               *CloudStorageConfig `json:"cloud_storage_config,omitempty"`
+	AdvanceTimeout                   *uint               `json:"advance_timeout,omitempty"`
```

Comment:
maybe it's better to add a time unit to the field,  something like AdvenceTiimeoutInSecond